airbyte-webapp/src/pages/connectorBuilder/ConnectorBuilderListPage/ConnectorBuilderListPage.tsx

```diff
@@ -0,0 +1,125 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { createColumnHelper } from "@tanstack/react-table";
+import classNames from "classnames";
+import queryString from "query-string";
+import { useCallback, useMemo } from "react";
+import { FormattedMessage } from "react-intl";
+import { useNavigate } from "react-router-dom";
+
+import { DefaultLogoCatalog } from "components/common/DefaultLogoCatalog";
+import { HeadTitle } from "components/common/HeadTitle";
+import { SortOrderEnum } from "components/EntityTable/types";
+import { Button } from "components/ui/Button";
+import { FlexContainer } from "components/ui/Flex";
+import { Heading } from "components/ui/Heading";
+import { NextTable } from "components/ui/NextTable";
+import { SortableTableHeader } from "components/ui/Table";
+import { Text } from "components/ui/Text";
+
+import { useQuery } from "hooks/useQuery";
+import { useListProjects } from "services/connectorBuilder/ConnectorBuilderProjectsService";
+import { useCurrentWorkspaceId } from "services/workspaces/WorkspacesService";
+
+import styles from "./ConnectorBuilderListPage.module.scss";
+import { ConnectorBuilderRoutePaths } from "../ConnectorBuilderRoutes";
+
+interface Project {
+  name: string;
+  version: string;
+}
+
+export const ConnectorBuilderListPage: React.FC = () => {
+  const navigate = useNavigate();
+  const query = useQuery<{ sortBy?: string; order?: SortOrderEnum }>();
+  const workspaceId = useCurrentWorkspaceId();
+  const projectsReadList = useListProjects(workspaceId);
+  // TODO: set version based on activeDeclarativeManifestVersion once it is added to the API
+  const projects = projectsReadList.projects.map((projectDetails) => ({ name: projectDetails.name, version: "draft" }));
+
```

Comment:
I wrote this down in a pretty convoluted way in the issue, but if there are no builder projects, it should redirect immediately to `connector-builder/create` (with `replace: true`) - right now it shows an empty table:
<img width="1278" alt="Screenshot 2023-02-23 at 12 06 57" src="https://user-images.githubusercontent.com/1508364/220889459-03661c53-22d2-4f3c-ac65-f84ccbf452fd.png">

Reply:
The e2e tests are currently failing, but I think fixing this issue would fix the tests as well

Reply:
Ah yeah I forgot to add this, will do